spec.md

```diff
@@ -74,7 +74,7 @@ The `Span` interface must have the following capabilities:
 - Create and start a new child `Span` with a given operation name. Any trace attributes must be passed through into the child. **(py: `start_child`, go: `StartChild`)**
 - Finish the (already-started) `Span`.  Finish should be the last call made to any span instance, and to do otherwise leads to undefined behavior. **(py: `finish`, go: `Finish`)**
 - Set a key:value tag on the `Span`. The key must be a `string`, and the value must be either a `string`, a `boolean`, or a numeric type. Behavior for other value types is undefined. If multiple values are set to the same key (i.e., in multiple calls), implementation behavior is also undefined. **(py: `set_tag`, go: `SetTag`)**
-- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). **(py: `log_event, log_event_with_payload`, go: `LogEvent, LogEventWithPayload`)**
+- Add a new log event to the `Span`, accepting an event name `string` and an optional structured payload argument. If specified, the payload argument may be of any type and arbitrary size, though implementations are not required to retain all payload arguments (or even all parts of all payload arguments). An optional timestamp can be used to specify a past timestamp. **(py: `log`, go: `Log`)**
```

Comment:
I was thinking that the "pseudocode" would correspond to the python API since it's actually python code. :)

Someday we can bother hooking up use-case examples in *all* officially supported platforms (ideally with a page-wide selector for example language), at which point the Go examples would indeed use `LogEvent` or similar. But I like the idea of having the pointer at this line reference `Log` since it's the superset.

PS: we could also get rid of the pointers entirely in the semantic spec... though I thought they were helpful to ground things in reality.